### DISCUSSION TITLE
Request sent every char inside input

### DISCUSSION THREAD

**Discussion by peroxovy** [livewire/livewire#6192] [7. Bugs (archived)] [answered]
### Livewire version

2.12.5

### Which PHP version are you using?

PHP 8.0

### Steps To Reproduce

Every time I type something inside an input like:

`<x-primary-text-input wire:model.lazy="form.name" name="name" id="name"
placeholder="ex. John Smith" required>
</x-primary-text-input>`

Browser send a request.

### Additional information

I've seen while using Livewire 2.12.5 that I cannot use `wire:model.lazy`, `wire:model.debouncing` or `wire:model.defer`.

Every time I'm typing into an input I see the request sent to a server.
Any thoughts how I can fix it?

Is there any way to refresh config maybe? 

I've published config file and assets after creating components. Can it be the root cause of problem?

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@peroxovy can you share what your text input blade component looks like?

**Reply by peroxovy** [reply]
This is the input component:

```
@props(['disabled' => false])

<input {{ $disabled ? 'disabled' : '' }} {!! $attributes->merge(['class' => 'bg-gray-100 focus:bg-gray-200 border-2 border-gray-400 text-gray-900 text-sm rounded-lg focus:ring-gray-200 focus:border-gray-200 block w-full p-2']) !!}>
```

It's the default breeze one.

**Reply by peroxovy** [reply]
To be more detailed, the things are done inside modal. The Modal is binded to a `bool` property, which hides/shows it on demand of user if it helps. 

Out of modal (when I put input and bind it to a property) it seems, that the `wire:model.lazy` etc. are working correctly.

**Comment by peroxovy** [comment, marked as answer]
Okay, I found the solution, but I also need an explanation to that.

The modal appears on demand, but... I need to bind it to a `bool` property, to know when it's shown or not.

The form is a child inside that binded let's say `div`, which do not allows to mark another `wire:model.lazy` etc. properties directly to form inputs.

I've deleted binding from parent div and it started to work... But why it doesn't work as I expected?
